scripts/gitlab_trigger_client_publish

```diff
@@ -63,10 +63,14 @@ for integ_version in $integration_versions; do
   all_repos=$($WORKSPACE/integration/extra/release_tool.py --list git)
 
   for repo in $all_repos; do
-    repo_version=$($WORKSPACE/integration/extra/release_tool.py \
+    v=$($WORKSPACE/integration/extra/release_tool.py \
                    --version-of $repo \
-                   --in-integration-version $integ_version |
-                   cut -d/ -f2);
+                   --in-integration-version $integ_version);
```

Comment:
Actually, a more correct fix would be to call `release_tool.py --list git --in-integration-version $integ_version` above. IMHO, we might think about doing that and reverting this fix, because it will mask real errors.

Reply:
ok no worries I can do it.

Reply:
`release_tool.py --list git --in-integration-version $integ_version`

@merlin-northern 

This still needs to be fixed.

`release_tool` will even give you JSON if you want.

Reply:
https://github.com/mendersoftware/mender-qa/pull/571